gnome-screenlock-password-opensuse132-20160620.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "type": "match",
+      "ypos": 237,
+      "height": 263,
+      "width": 458,
+      "xpos": 290
+    }
+  ],
+  "tags": [
+    "gnome-screenlock-password"

```

Comment:
Why this blacklisting and not whitelisting? Can this be done automatically for all ENV variables?